templates/cassiopeia/scss/offline.scss

```diff
@@ -5,8 +5,8 @@
 @import "tools/tools";
 
 html {
-  background: radial-gradient(circle, $gray-100, $gray-200);
-  background-color: $cyan;
+  background: radial-gradient(circle, gray(gray-100), gray(gray));
+  background-color: color(cyan);
```

Comment:
I dont understand this syntax but where you used it for white it was wrapped in " quotes

Reply:
Syntax corrected in last commit. But the offline page needs some love ;-)